### PR TITLE
asserts: remove/disable comma separated lists and their uses

### PR DESCRIPTION
**Pull request by pedronis**

### CODE REVIEW

**Comment by pedronis** [comment]
retest this please

File: asserts/device_asserts.go [hunk 1]
```diff
@@ -136,8 +128,8 @@ func assembleModel(assert assertionBase) (Assertion, error) {
 	// ignore extra headers and non-empty body for future compatibility
 	return &Model{
 		assertionBase: assert,
-		allowedModes:  allowedModes,
-		requiredSnaps: requiredSnaps,
+		allowedModes:  nil, // XXX: empty for now
```
Comment (chipaca) [inline asserts/device_asserts.go]: if these are going to be assumed empty for now wouldn't it behoove us to warn if it's set to something non-empty?

**Comment by chipaca** [comment]
:+1: with a Q

File: asserts/device_asserts.go (hunk 1 above)
Comment (pedronis) [inline asserts/device_asserts.go]: well the assertion is not even finalized, it's all broken/in flux  atm

**Comment by mvo5** [comment]
:+1: Curious if we have a direction how this field is going to look like? But lets keep out of 2.11 for now.

**Pull request merged**